src/main/java/im/toduck/domain/diary/domain/usecase/DiaryUseCase.java

```diff
@@ -81,10 +83,25 @@ private boolean isDiaryOwner(final Diary diary, final User user) {
 		return diary.isOwner(user);
 	}
 
-	@Transactional
+	@Transactional(readOnly = true)
 	public List<DiaryResponse> getDiariesByMonth(final Long userId, final int year, final int month) {
 		User user = userService.getUserById(userId)
 			.orElseThrow(() -> CommonException.from(ExceptionCode.NOT_FOUND_USER));
 		return diaryService.getDiariesByMonth(userId, year, month);
 	}
+
+	@Transactional(readOnly = true)
+	public MonthDiaryResponse getDiaryCountByMonth(Long userId, int year, int month) {
+		User user = userService.getUserById(userId)
+			.orElseThrow(() -> CommonException.from(ExceptionCode.NOT_FOUND_USER));
+		int lastMonth = month - 1;
+		int lastYear = year;
+		if (lastMonth == 0) {
+			lastMonth = 12;
+			lastYear--;
+		}
+		int thisMonthCount = diaryService.getDiaryCountByMonth(userId, year, month);
+		int lastMonthCount = diaryService.getDiaryCountByMonth(userId, lastYear, lastMonth);
```

Comment:
```suggestion
        // 현재 월 생성
        YearMonth currentMonth = YearMonth.of(year, month);
        
        // 이전 월 계산 (자동으로 연도 변경 처리)
        YearMonth previousMonth = currentMonth.minusMonths(1);
        
        int thisMonthCount = diaryService.getDiaryCountByMonth(userId, currentMonth.getYear(), currentMonth.getMonthValue());
        int lastMonthCount = diaryService.getDiaryCountByMonth(userId, previousMonth.getYear(), previousMonth.getMonthValue());
```

p5: 이렇게도 표현할 수 있을 것 같아요! 지금 방법도 충분히 좋은 것 같으니, 편하신 쪽으로 선택하시면 될 것 같습니다.